echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/slack/SlackNotificationService.groovy

```diff
@@ -62,8 +63,7 @@ class SlackNotificationService implements NotificationService {
           new SlackAttachment(subject, text, (InteractiveActions)notification.interactiveActions),
           address, true)
       }
-      log.trace("Received response from Slack: {} {} for message '{}'. {}",
-        response?.status, response?.reason, text, response?.body)
+      log.trace("Received response from Slack: {}", response?.string())
```

Comment:
Does this show a gap in Retrofit2SyncCall that the entire Response is no longer available...only the ResponseBody?  I suppose if we get here there's no exception, so we know the response code is 2xx...and since this is a trace message it seems OK to drop that info.